tests/collaborative/ot/ot_rows_added.test.ts

```diff
@@ -97,6 +101,34 @@ describe("OT with ADD_COLUMNS_ROWS with dimension ROW", () => {
     });
   });
 
+  describe.each(OT_TESTS_ZONE_DEPENDANT_COMMANDS)("zone dependant commands", (cmd) => {
+    test(`add rows after ${cmd.type}`, () => {
+      const command = { ...cmd, sheetId, zone: toZone("A1:C1") };
+      const result = transform(command, addRowsAfter);
+      expect(result).toEqual(command);
+    });
+    test(`add rows after ${cmd.type}`, () => {
+      const command = { ...cmd, sheetId, zone: toZone("A10:B11") };
+      const result = transform(command, addRowsAfter);
+      expect(result).toEqual({ ...command, zone: toZone("A12:B13") });
+    });
+    test(`add rows after in ${cmd.type}`, () => {
+      const command = { ...cmd, sheetId, zone: toZone("A5:B6") };
+      const result = transform(command, addRowsAfter);
+      expect(result).toEqual({ ...command, zone: toZone("A5:B6") });
+    });
+    test(`add rows before in ${cmd.type}`, () => {
```

Comment:
those two tests are actually the same. I guess you were trying to test row addition inside a group (and the group should grow)